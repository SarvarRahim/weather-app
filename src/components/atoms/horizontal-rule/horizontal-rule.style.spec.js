const { customRender } = require('@test-utils/helpers');
const { HorizontalRule } = require('./horizontal-rule.style');

describe('Atoms Context', () => {
  test('Should render HorizontalRule component', async () => {
    const { asFragment } = customRender(<HorizontalRule />);
    expect(asFragment()).toMatchSnapshot();
  });
});
