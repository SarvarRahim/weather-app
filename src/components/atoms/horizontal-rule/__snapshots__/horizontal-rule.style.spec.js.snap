// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms Context Should render HorizontalRule component 1`] = `
<DocumentFragment>
  .c0 {
  height: 0.1rem;
  margin-top: 2.4rem;
  margin-bottom: 2.4rem;
  background-color: #dadada;
}

<div
    class="c0"
  />
</DocumentFragment>
`;
